lib/uplink/bucket.go

```diff
@@ -96,6 +96,38 @@ type UploadOptions struct {
 func (b *Bucket) UploadObject(ctx context.Context, path storj.Path, data io.Reader, opts *UploadOptions) (err error) {
 	defer mon.Task()(&ctx)(&err)
 
+	upload, err := b.NewWriter(ctx, path, opts)
```

Comment:
is all that changed in this file function reordering? or did something else change?

Reply:
there are `NewWriter` and `NewReader` for `bucket`